packages/wdio-sumologic-reporter/src/index.ts

```diff
@@ -35,80 +47,78 @@ export default class SumoLogicReporter extends WDIOReporter {
         this.isSynchronising = false
 
         this.errorCount = 0
-        this.specs = {}
-        this.results = {}
         this.interval = setInterval(this.sync.bind(this), this.options.syncInterval)
     }
 
     get isSynchronised() {
         return this.unsynced.length === 0
     }
 
-    onRunnerStart(runner) {
+    onRunnerStart(runner: string) {
         this.unsynced.push(stringify({
             time: dateFormat(new Date(), DATE_FORMAT),
             event: 'runner:start',
             data: runner
         }))
     }
 
-    onSuiteStart(suite) {
+    onSuiteStart(suite: WDIOReporter.Suite | string) {
```

Comment:
Why do we need a `| string` is this and the other functions?

Reply:
https://github.com/webdriverio/webdriverio/pull/5998/files/a5e271c6e876156f1a283b926dacedc7ea5b9a68#diff-d3bd55551195e039e0aedeec34687f7aecd889f49af2b15bdea851ac705fe08eL36

In the above step we are passing arguments as string, hence i have added | string in the function

Reply:
GitHub doesn't highlight the line, can you say which file and line number?

Reply:
`reporter.onTestStart('onTestStart')`
The above line(Line number:40 ) in "reporter.test.ts " file .

Reply:
@suniljaiswal01 the test is wrong here. All of these listeners take objects.

Reply:
Will update the test then